README.md

```diff
@@ -3627,22 +3627,50 @@ resource cleanup when possible.
 <sup>[[link](#percent-s)]</sup>
 
 * <a name="percent-literal-braces"></a>
-  Prefer `()` as delimiters for all `%` literals, except `%r`. Since parentheses
-  often appear inside regular expressions in many scenarios a less common
-  character like `{` might be a better choice for a delimiter, depending on the
-  regexp's content.
-<sup>[[link](#percent-literal-braces)]</sup>
+  Prefer brackets as delimiter for all `%` literals, because their usage does 
```

Comment:
Let's make the wording compatible with the previous version of the rule. E.g. you can add that `[]` delimiters are ok when you're dealing with array literals.